content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,476 @@
+title: NeuVector Prime in Azure: Usage Instructions
+---
+theme: suse
+---
+body:
+
+##  How to install NeuVector and configure Billing-adapter
+
+The NeuVector offer handles billing by gathering usage data from NeuVector. For complete installation, Ensure to follow all the steps
+till NeuVector Usage Record and Billing.
+
+##Prerequisites
+
+These instructions assume you have an AKS cluster running as per the [NeuVector System requirements document](https://open-docs.neuvector.com/basics/requirements)
+
+
+A step by step walk-through for creating a new deployment of NeuVector from the Azure Marketplace Overview page.
+
+1. Choose the plan from the dropdown list. ( "Plans + Pricing" tab more details about Software plan)
+2. Click 'Create' 
+
+## Basics
+
+1. Select existing subscription from the dropdown list.
+2. Select existing Resource group from the dropdown list.
+3. Select existing AKS Cluster Name from the dropdown list.
+4. Choose name for Cluster extension resource name as it suggested by the 
+   tool tip "Allow alphanumeric, and dots. And the value must be 1-253 characters long". 
```

Comment:
tool tip content between "" is copy paste from Azure market
place..

Reply:
Ugh :-/  sounds like ideally it would be fixed there too... we shouldn't start a sentence with "And"... it should probably read "Allows alphanumeric, dots, and the value must be between 1-253 characters long"

Reply:
Fixed documentation, How do we fix the tooltip from Azure Marketplace ?

Reply:
I can fix it in the bundle. IIRC that was text given to me directly by Selvam based on somewhere else they used it or such

```
	"toolTip": "Only use alphanumeric and dots. The value must be 1-253 characters long.",
```

one can argue if we should make this bigger than 1 as a minimum,. Originally we had 12.  @guangyee what did you do for Rancher?

Reply:
> Fixed documentation, How do we fix the tooltip from Azure Marketplace ?

I already fixed it. We just need to decide on the min length. It is not a password field, so I will set it back to 8 unless @guangyee has any objections